rhoas-client/src/main/java/com/redhat/service/bridge/rhoas/dto/TopicAndServiceAccountRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.service.bridge.rhoas.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TopicAndServiceAccountRequest {
```

Comment:
Same here

Reply:
This is part of RhoasClient, which is an aggregated client for our specific needs, so I don't expect this to be part of the SDK.